packages/integration/src/lib/map/advanced-map-tool/advanced-coordinates/advanced-coordinates.component.ts

```diff
@@ -212,6 +197,39 @@ export class AdvancedCoordinatesComponent implements OnInit, OnDestroy {
     return coord;
   }
 
+  updateCoordinates() {
+    this.setScaleValue(this.map);
```

Comment:
Il n'y a pas de lien entre l'update de la coordonnée et le scale. Je déplacerais cette méthode dans le subscribe du `this.map.viewController.state$`